common/components/componentsBySection/FindProjects/ProjectCardsContainer.jsx

```diff
@@ -94,10 +94,21 @@ class ProjectCardsContainer extends React.Component<{||}, State> {
 
   _renderPagination(): ?React$Node {
     if (this.state.current_page === this.state.project_pages) {
-      return null; // don't render button if we've loaded the last page
+      return (
```

Comment:
I don't know that we should show the greyed-out button when there are no more results.  It could give users the mistaken impression that there are more projects waiting to load.

Reply:
Agreed.  I've return it to showing nothing when all projects have been loaded.